utils/build-script-impl

```diff
@@ -1546,32 +1767,32 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_COMPILE_TOOLS_DEPLOYMENT_TARG
 
                 cmake_options=(
                     "${cmake_options[@]}"
-                    -DCMAKE_INSTALL_PREFIX:PATH="${INSTALL_PREFIX}"
-                    -DLLVM_CONFIG:PATH="$(build_directory_bin ${deployment_target} llvm)/llvm-config"
+                    -DCMAKE_INSTALL_PREFIX:PATH="$(get_host_install_prefix ${host})"
```

Comment:
Yes, this is exactly right, and it is what I'm seeing.  So the issue is either somewhere else, or it is my local misconfiguration.  See my other comment -- I'm running another build.